### ISSUE TITLE
Feature request

### ISSUE THREAD

**Issue by james-shulman** [dorzki/Slack-Notifications#53]
Any chance getting new notifications for:

‘Author’ User Role Login
‘Contributor’ User Role Login
‘Editor’ User Role Login
Custom User Role(s)/Capabilities (these can be created using for example User Role Editor plugin)

Adding support for the roles above and optionally other roles (custom roles and/or capabilities created) would be very useful for tracking and accountability purposes as well as beneficial for security.

Basically, expanding the user’s choice of other roles and or capabilities as well and not just notify on ‘Administrator’ role.  This can be in a form of a comma separated list/textbox in the notification.

Thank You,

https://wordpress.org/plugins/dorzki-notifications-to-slack/

**Comment by DorZuberi** [comment]
Hi James,
I'm working on a new a much improved plugin version.

I've created dev branch, you can keep track there on my progress, I hope to have it ready by the end of June.
